crates/wash/src/cli/config.rs

```diff
@@ -84,6 +116,221 @@ impl CliCommand for ConfigCommand {
                     Some(serde_json::to_value(&config).context("failed to serialize config")?),
                 ))
             }
+            ConfigCommand::Cleanup {
+                config,
+                cache,
+                data,
+                all,
+                dry_run,
+            } => {
+                let config_dir = ctx.config_dir();
+                let cache_dir = ctx.cache_dir();
+                let data_dir = ctx.data_dir();
+
+                let mut cleanup_paths: Vec<PathBuf> = Vec::new();
+
+                if !(*config || *cache || *data || *all) {
+                    return Ok(CommandOutput::error(
+                        "Nothing to do. Please specify --config, --data, --cache, or --all to clean up.",
+                        Some(serde_json::json!({
+                            "message": "Nothing to do. Please specify --config, --data, --cache, or --all to clean up.",
+                            "success": true,
+                        })),
+                    ));
+                }
```

Comment:
I think we could use the clap ArgGroup capability to put `config`, `data`, `cache`, and `all` in a single arg group, make the group required, and make multiple flags `true`. That would prevent us from having to update this if we add more fields in the future!